packages/core/data-transfer/src/strapi/providers/local-destination/__tests__/assets.test.ts

```diff
@@ -45,6 +45,7 @@ const strapiFactory = getStrapiFactory({
 
 describe('Local Strapi Destination Provider - Get Assets Stream', () => {
   test('Throws an error if the Strapi instance is not provided', async () => {
+    /* eslint-disable-next-line @typescript-eslint/ban-ts-comment */
```

Comment:
Why have you had to add these to the PR? I think this is OOS? does the linter fail?

Reply:
There is a type error in my editor and I saw it was because of this comment missing

Reply:
I am gonna remove it because it is out of scope